### ISSUE TITLE
production ready?

### ISSUE THREAD

**Issue by cloudcompute** [cloudcreate-dk/essentials-project#58]
Hi 

Is this project in production, any one using it Or is it intended for demo purpose only?

Thank you

**Comment by cloudcreate-dk** [comment]
Thanks for the question.
The project is still in pre-release and currently undergoing intensive development efforts, which means that many features are being worked on in parallel, the API’s are still subject to change.
Certain modules, such as the types-springdata-jpa and kotlin-eventsourcing, are still highly experimental and unstable and may be removed in the future or merged into other modules.
Users curious about the project are welcome to explore the project's capabilities and provide feedback, but should be aware of instabilities, updates and changes that could impact its use in specific scenarios.

**Comment by cloudcompute** [comment]
Ok thanks.
